core/app/models/concerns/spree/filter_param.rb

```diff
@@ -0,0 +1,21 @@
+module Spree
+  module FilterParam
+    extend ActiveSupport::Concern
+
```

Comment:
Had to drop `attr_writer :filter_param` in here to get the missing migrations to run.

Reply:
```
matt@Matthews-iMac teee % rails db:migrate

== 20210505150238 AddParamToSpreeProductProperties: migrating =================
-- column_exists?(:spree_product_properties, :param)
   -> 0.0035s
-- add_column(:spree_product_properties, :param, :string)
   -> 0.0017s
-- add_index(:spree_product_properties, :param)
   -> 0.0457s
rails aborted!
StandardError: An error has occurred, this and all later migrations canceled:

undefined method `filter_param=' for #<Spree::ProductProperty id: 1, value: "Wannabe", product_id: 12, property_id: 1, created_at: "2021-05-05 14:57:58.403270000 +0000", updated_at: "2021-05-05 14:58:01.535395000 +0000", position: 1, show_property: true, param: nil, preferences: nil>
/Users/matt/spark/teee/db/migrate/20210505150238_add_param_to_spree_product_properties.spree.rb:9:in `change'
/Users/matt/spark/teee/bin/rails:5:in `<top (required)>'
/Users/matt/spark/teee/bin/spring:10:in `block in <top (required)>'
/Users/matt/spark/teee/bin/spring:7:in `<top (required)>'

Caused by:
NoMethodError: undefined method `filter_param=' for #<Spree::ProductProperty id: 1, value: "Wannabe", product_id: 12, property_id: 1, created_at: "2021-05-05 14:57:58.403270000 +0000", updated_at: "2021-05-05 14:58:01.535395000 +0000", position: 1, show_property: true, param: nil, preferences: nil>
/Users/matt/spark/teee/db/migrate/20210505150238_add_param_to_spree_product_properties.spree.rb:9:in `change'
/Users/matt/spark/teee/bin/rails:5:in `<top (required)>'
/Users/matt/spark/teee/bin/spring:10:in `block in <top (required)>'
/Users/matt/spark/teee/bin/spring:7:in `<top (required)>'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)
matt@Matthews-iMac teee %
```

Reply:
Upgrading from a fresh install from the current master to this PR.